lib/average.js

```diff
@@ -0,0 +1,44 @@
+function total(arr) {
+  return arr.reduce(function(a, b) { return a + b; }, 0);
+}
+
+function mean(arr) {
+  var total = 0;
+  for(var i = 0; i < arr.length; i++) {
+    total += arr[i];
+  }
+  return total / arr.length;
+}
+
+function median(arr) {
+  sorted = arr.sort(function(a, b) {
```

Comment:
Remember to declare variables with `var`. Otherwise, you're accidentally creating dangerous global variables